package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "Boilerplate",
+  "authors": [
+    "Jeff Lupinski <iam@jefflupinski.com>",
+    "Seth Krasnianski <seth@sjkstudios.com>"
+  ],
+  "version": "0.0.1",
+  "description": "Client-side boilerplate. Made with gulp and friends.",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:schwill/boilerplate.git"
+  },
+  "scripts": {
+    "start": "gulp",
+    "build": "NODE_ENV=production gulp build",
+    "clean": "rm -rf public"
+  },
+  "devDependencies": {
+    "babel": "^5.8.23",
```

Comment:
Suggest using babel-core, otherwise returns Failed to load external module babel-core/register